tools/plotly_regression_performance_plots/plot_regression_performance.py

```diff
@@ -62,7 +69,7 @@ def main(infile_input, infile_output):
     )
 
     layout_true_pred = go.Layout(
-        title='True vs predicted values',
+        title='True vs predicted values (RMSE: ' + str(np.round(rmse, 2)) + ', R2: ' + str(np.round(r2, 2)) + ')',
```

Comment:
You should use string substitutions like `"%s (RMSE: 766) %s (foo bar)" % (var1, var2)`

Reply:
fixed!